lsp-haskell.el

```diff
@@ -263,6 +308,26 @@ and `lsp-haskell-server-args' and `lsp-haskell-server-wrapper-function'."
     ;; This is somewhat irrelevant, but it is listed in lsp-language-id-configuration, so
     ;; we should set something consistent here.
     :language-id "haskell"
+    :download-server-fn (lambda (_client callback error-callback _update?)
+			  (lsp-package-ensure
+			   'haskell-language-server-wrapper
+			   (lambda ()
+			     (let*
```

Comment:
Maybe a named function `install-server :: Version -> IO ()` would be good, might be independently useful (users can call it directly if they want).